include/aktualizr-lite/api.h

```diff
@@ -15,6 +15,11 @@
 class Config;
 class LiteClient;
 
+namespace Uptane {
+class Target;
```

Comment:
We shouldn't expose the aktualizr's internal types in the interface.

Reply:
Fixed